src/driftpy/math/exchange_status.py

```diff
@@ -0,0 +1,18 @@
+from typing import Union
+from driftpy.types import ExchangeStatus, PerpMarketAccount, SpotMarketAccount, StateAccount, is_one_of_variant
+
+
+def exchange_paused(state: StateAccount) -> bool:
+    return state.exchange_status != ExchangeStatus.ACTIVE
+
+def fill_paused(state: StateAccount, market: Union[PerpMarketAccount, SpotMarketAccount]) -> bool:
+    return (
+        (state.exchange_status & ExchangeStatus.FILL_PAUSED) == ExchangeStatus.FILL_PAUSED or
+        is_one_of_variant(market.status, ['PAUSED', 'FILL_PAUSED'])
```

Comment:
dont these need to be Paused, FillPaused?

Reply:
think we need to update the enums

Reply:
Yeah I'll update all of the enums today to all use uniform anchor-style casing